.rubocop.yml

```diff
@@ -32,9 +32,6 @@ Metrics/PerceivedComplexity:
 Naming/AccessorMethodName:
   Enabled: false
 
-Security/Open:
```

Comment:
reason, http_persistent?

Reply:
Not related.

- We don't use open anywhere
- I don't like having security-related issues ignored

Reply:
> I don't like having security-related issues ignored

this is the mindset 💯

Reply:
> We don't use open anywhere

why we ignored it before is a question though